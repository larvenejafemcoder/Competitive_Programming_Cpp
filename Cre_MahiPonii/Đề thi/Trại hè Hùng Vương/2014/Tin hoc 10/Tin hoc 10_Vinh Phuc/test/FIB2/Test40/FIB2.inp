100
12 221
0 1
39 13043141
1 1
39 5049268
14 246
18 3100
34 1545171
23 19593
32 457853
9 22
23 5461
34 162869
31 1032291
36 5402019
38 60920397
20 8499
22 12754
25 105822
26 145402
8 34
26 141821
36 5230440
40 27698741
36 23169201
36 251044
38 28134301
33 527223
1 1
0 1
23 15152
14 264
3 1
44 303209406
24 21931
20 5885
11 47
40 35713073
3 1
9 45
4 2
20 10254
2 2
6 8
14 204
10 15
39 45959741
12 23
21 6831
21 15098
13 155
40 18696823
42 368617175
43 288742242
31 1152295
44 742442738
13 258
38 17842140
22 9009
12 110
17 158
39 73182719
39 2339651
43 329238095
35 4190998
26 154545
4 4
20 5120
31 1720471
10 49
7 8
37 1724930
6 4
4 2
31 1826807
28 336354
11 26
12 177
34 6298210
25 97856
10 43
14 338
41 15095023
38 33401931
10 51
16 487
16 494
17 1886
32 1332075
2 1
5 6
19 2049
35 5775375
0 1
13 253
41 72982815
10 7
34 2999947
33 968164
17 1268
