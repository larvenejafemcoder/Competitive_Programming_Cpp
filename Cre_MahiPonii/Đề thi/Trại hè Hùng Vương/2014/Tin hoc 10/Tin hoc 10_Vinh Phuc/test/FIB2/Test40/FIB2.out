85
1
4982037
0
1928649
94
1184
590203
7484
174884
9
2086
62211
394300
2063388
23269521
3246
4872
40421
55539
13
54171
1997850
10579978
8849847
95890
10746347
201381
0
1
5788
101
0
115815688
8377
2248
18
13641180
0
17
1
3917
1
3
78
6
17555059
9
2609
5767
59
7141551
140799232
110289723
440138
283587891
99
6815091
3441
42
60
27953311
893667
125757762
1600819
59031
2
1956
657162
19
3
658865
2
1
697778
128476
10
68
2405702
37378
17
129
5765786
12758402
20
186
189
721
508807
1
2
783
2205997
1
97
27876955
3
1145878
369806
484
